### ISSUE TITLE
Allow for adding calendar entries to a contact for holidays

### ISSUE THREAD

**Issue by allo-** [monicahq/monica#2746]
**Is your feature request related to a problem? Please describe.**

It can be hard to keep up with who is making holidays and not available for events. It would be nice, if you could add the information when somebody is on holiday and maybe display it as a Gantt Chart.

**Describe the solution you'd like**

- Add Timespans with Description and Category in the contact view, e.g. "Summer Holiday" (tagged: holiday, available: false)
- Possibly allow to link peoples activities together (x and y are making holidays together)
- Show it the overview and maybe as a Gantt Chart

Basic planning (like doodle) would be a nice idea, but for a start it would be just good to store and visualize the information.

**Comment by tbirrell** [comment]
So you want to be able to add known upcoming events?

**Comment by allo-** [comment]
Yes, but not events like "birthday", but events like "holiday from 01.10.2019 - 31.10.2019, unavailable", so I can see with a quick glance on the overview page that someone will be unavailable for our regular meeting tomorrow.
Views like a gantt chart of all people, groups or tags could allow for planning meetings (i.e. telling others "lets meet at (date), because that's the only date in the next time when nobody is on holiday).

A simple implementation would be to allow adding a timespan span (and maybe a repetition option, like "she's always dancing on mondays") as reminder and showing it in the events view like the single day events.

**Comment by tbirrell** [comment]
The calander/gnatt chart side of this seems a little out of scope for Monica. Though I quite like the idea of tracking repetitive events that I'm not a part of, which is something that falls under what Monica tries to help with.

**Comment by allo-** [comment]
The chart would be a nice visualization, but the most simple use case is, when you want to meet people and forgot that they are on holiday.
More simple visualizations include a tag "unavailable until ..." next to the name, an entry in the upcoming events list or a list "people currently on holiday".
